src/powers.csv

```diff
@@ -1,8 +1,8 @@
 "Name","Class","Type","Level","Gold","Energy","Raw Description","Icon Link","Phase","Dependant on","Attributes Impacted","Effect","Spot Colour","Spot Amount","Max Spot amount"
 "Thorns","druid","active","3","8","3","<i>End of your Reroll step:</i></p>Remove up to {dice green 3 5+} to place an equal number of hit tokens in the damage box","https://wow.zamimg.com/images/wow/icons/large/spell_nature_thorns.jpg","End_Reroll","","damage 0 1","spot: { colour: ['green'], min: 5, maxToSpot: 3, attributesIncreased: ['damage'] }","Green","5+","3"
-"Entangling Roots","druid","instant","2","6","1","<i>End of your Reroll step:</i></p>Spot {dice green,red 1 7+} to roll an additional blue dice and gain <b>ATTRITION + 1<b/>","https://wow.zamimg.com/images/wow/icons/large/spell_nature_stranglevines.jpg","End_Reroll","","dice_self_blue 0 1, attrition 0 1","spot: { colour: ['green', 'red'], minTarget: 7, maxToSpot: 1, attributesIncreased: ['dice_self_blue', attrition] }","Red, Blue","7+","1"
+"Entangling Roots","druid","instant","2","6","1","<i>End of your Reroll step:</i></p>Spot {dice green,red 1 7+} to roll an additional blue dice and gain <b>ATTRITION + 1</b>","https://wow.zamimg.com/images/wow/icons/large/spell_nature_stranglevines.jpg","End_Reroll","","dice_self_blue 0 1, attrition 0 1","spot: { colour: ['green', 'red'], minTarget: 7, maxToSpot: 1, attributesIncreased: ['dice_self_blue', attrition] }","Red, Blue","7+","1"
 "Demoralizing Roar","druid","instant","1","2","1","You must have <b>Bear Form</b> equipped to use this power. </p><i>Start of your Dice Pool step:</i></p>Add {dice green 2} to your dice pool","https://wow.zamimg.com/images/wow/icons/large/ability_druid_demoralizingroar.jpg","Start_Dice","Bear Form","dice_self_green 2","attributeChange: {  type: 'increase', amount: 2, attribute:'dice_self_green' }","","",""
-"Insect Swarm","druid","instant","3","7","3","<i>Start of your Dice Pool step:</i></p>Add {dice green 2} to your dice pool and gain <b>ATTRITION + 2<//b>","https://wow.zamimg.com/images/wow/icons/large/spell_nature_insectswarm.jpg","Start_Dice","","dice_self_green 2, attrition 2","[ { attributeChange: { amount: 2, type: 'increase', attribute: 'dice_self_green' } }, { attributeChange: { type: 'increase', amount: 2, attribute: 'attrition' } } ","","",""
+"Insect Swarm","druid","instant","3","7","3","<i>Start of your Dice Pool step:</i></p>Add {dice green 2} to your dice pool and gain <b>ATTRITION + 2</b>","https://wow.zamimg.com/images/wow/icons/large/spell_nature_insectswarm.jpg","Start_Dice","","dice_self_green 2, attrition 2","[ { attributeChange: { amount: 2, type: 'increase', attribute: 'dice_self_green' } }, { attributeChange: { type: 'increase', amount: 2, attribute: 'attrition' } } ","","",""
```

Comment:
@ThatRanga already spotted 2 data entry errors i humbly request that you visit dovetailapp.com/company and ponder the values
<img width="1246" alt="Screen Shot 2022-05-23 at 12 23 04 am" src="https://user-images.githubusercontent.com/2404098/169700189-a597affe-b121-43e1-871d-06c45ca5b370.png">

Reply:
Interestingly I checked the sheet and found that you hadn't updated it in our source of truth for data which means now we have inconsistent data. I humbly requests that you also visit dovetailapp.com/company and ponder the values
![image](https://user-images.githubusercontent.com/103867191/169720378-68058a35-8933-433b-a570-05185356fffd.png)